linters/pyright/test_data/pyright_v1.1.304_basic.check.shot

```diff
@@ -0,0 +1,135 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Testing linter pyright test basic 1`] = `
+{
+  "issues": [
+    {
+      "code": "reportGeneralTypeIssues",
+      "column": "57",
+      "file": "test_data/basic.in.py",
+      "issueUrl": "http://pyright-messages.wikidot.com/messages:reportGeneralTypeIssues",
+      "level": "LEVEL_HIGH",
+      "line": "1",
+      "linter": "pyright",
+      "message": ""Enum" is unknown import symbol",
+      "targetType": "python",
+    },
+    {
+      "column": "13",
+      "file": "test_data/basic.in.py",
+      "issueUrl": "http://pyright-messages.wikidot.com/messages:",
```

Comment:
Where'd you get the template for issueURL? Looks like these aren't resolving correctly.

Reply:
Fixed. I did a crappy /pylint/pyright/ replacement and didn't fix this.